Package.swift

```diff
@@ -4,30 +4,23 @@ import PackageDescription
 
 let package = Package(
     name: "mParticle-Kochava",
-    platforms: [
-            .iOS("10.3"),
-            .tvOS("10.2"),
-        ],
+    platforms: [ .iOS("10.3") ],
```

Comment:
did you mean to remove tvOS? I see Kochava's SDK supports it - and I can connect a tvOS mParticle input to Kochava...